hometask_3.py

```diff
@@ -0,0 +1,45 @@
+# Task 1
+
+print(' I love python! ' * 42)
+
+# =================================================================================
+
+#Task 2
+age_in_month = 31 * 12 + 9
+print("My age = " + str(age_in_month) + " months")
+
+# =================================================================================
+
+#Task 3
+
+age_in_years = age_in_month / 12
```

Comment:
The value here is float. For future usage in text it is better to transform the value to int, using int() or //

Reply:
Ok, i get it